<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnUV4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>更改UV4.EXE路径</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtUV4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPrj">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrj">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>更改项目路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompile">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>编译并烧录</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtCompileResult">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
